modules/imgproc/src/imgwarp.cpp

```diff
@@ -3226,24 +3228,7 @@ void cv::resize( InputArray _src, OutputArray _dst, Size dsize,
         resizeArea_<double, double>, 0
     };
 
-    Size ssize = _src.size();
```

Comment:
please, restore the original semantics; above it's explained why it is important